docker/.env

```
@@ -543,3 +545,53 @@ SMQ_GRAFANA_ADMIN_PASSWORD=supermq
+MANAGER_CLIENT_ID=""
+MANAGER_CLIENT_KEY=""
+MANAGER_HTTP_HOST=manager
+MANAGER_HTTP_PORT=7070
+MANAGER_HTTP_SERVER_CERT=""
+MANAGER_HTTP_SERVER_KEY=""
+MANAGER_OTEL_URL=${SMQ_JAEGER_URL}
+MANAGER_TRACE_RATIO=${SMQ_JAEGER_TRACE_RATIO}
+
+## Proplet
+PROPLET_LOG_LEVEL=debug
+PROPLET_INSTANCE_ID=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)